qemu/tests/vioinput_hotplug.py

```diff
@@ -0,0 +1,77 @@
+import logging
+import time
+
+from virttest import error_context
+from virttest.qemu_devices import qdevices
+from provider import input_tests
+from avocado.core import exceptions
```

Comment:
Should not using this directly in test scripts, instead, please use `test.fail`/`test.error`...

Reply:
ok, will update~